pkg/client/retrieve.go

```diff
@@ -40,13 +40,22 @@ var tarCommand = []string{
 	fmt.Sprintf("tar cf - %s/*.tar.gz", config.MasterResultsPath),
 }
 
-func (c *SonobuoyClient) RetrieveResults(cfg *RetrieveConfig) (io.Reader, <-chan error) {
+func (c *SonobuoyClient) RetrieveResults(cfg *RetrieveConfig) (io.Reader, <-chan error, error) {
```

Comment:
nit: I know this function didn't have a comment before, but since it is an exported method we should add one. Especially since we are modifying the signature. We should make it clear what the channel of errors is for and what the "raw" error is for.

Reply:
Agreed. I'll add a comment here.